command_line_assistant/history.py

```diff
@@ -13,7 +13,7 @@ def handle_history_read(config: Config) -> dict:
 
     filepath = config.history.file
     if not filepath or not filepath.exists():
-        logging.warning(f"History file {filepath} does not exist.")
+        logging.warning("History file %s does not exist.", filepath)
```

Comment:
What's the reason for moving away from f-strings here?

Reply:
Fixing ruff/pyright linting warning for lazy logging

Reply:
Interesting.

Reply:
I believe this is [G004](https://docs.astral.sh/ruff/rules/logging-f-string/). TIL. Are we going to enable that in the `pyproject.toml`?

Reply:
We can. I will add it in another PR.